plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -623,7 +657,8 @@ class GesturesPluginTest {
   }
 
   @Test
-  fun verifyRotate() {
+  fun verifyRotateWithHighLevelAnimation() {
```

Comment:
let's rename test to identify that in this case `simultaneousRotateAndPinchToZoomEnabled = false`